components/script/dom/performanceresourcetiming.rs

```diff
@@ -0,0 +1,232 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+use dom::bindings::codegen::Bindings::PerformanceBinding::DOMHighResTimeStamp;
+use dom::bindings::codegen::Bindings::PerformanceResourceTimingBinding::{self, PerformanceResourceTimingMethods};
+use dom::bindings::inheritance::Castable;
+use dom::bindings::root::{Dom, DomRoot};
+use dom::bindings::str::DOMString;
+use dom::bindings::num::Finite;
+use dom::bindings::reflector::{Reflector, reflect_dom_object};
+use dom::globalscope::GlobalScope;
+use dom::performanceentry::PerformanceEntry;
+use dom::window::Window;
+use dom_struct::dom_struct;
+use servo_url::ServoUrl;
+
+// TODO UA may choose to limit how many resources are included as PerformanceResourceTiming objects
+// recommended minimum is 150, can be changed by setResourceTimingBufferSize in performance
+// https://w3c.github.io/resource-timing/#sec-extensions-performance-interface
+// 
+// TODO Cross origin resources MUST BE INCLUDED as PerformanceResourceTiming objects
+// https://w3c.github.io/resource-timing/#sec-cross-origin-resources
+
+#[dom_struct]
+pub struct PerformanceResourceTiming {
+    reflector_: Reflector,
+	entry: Dom<PerformanceEntry>,
```

Comment:
This should be `performanceentry: PerformanceEntry` instead of the `reflector_` and `entry` members, since this interface inherits from PerformanceEntry ([docs](https://doc.servo.org/script/dom/index.html#inheritance)).